frontend/common/src/components/Pill/Pill.tsx

```diff
@@ -52,23 +56,38 @@ const colorMap: Record<
   },
 };
 
+const sizeMap: Record<PillSize, Record<string, string>> = {
+  sm: {
+    "data-h2-font-size": "base(0.6rem)",
+    "data-h2-padding": "base(x.1, x.25)",
+  },
+  md: {
+    "data-h2-padding": "base(x.25, x.75)",
+    "data-h2-font-size": "base(caption)",
```

Comment:
Just to make it consistent with the other sizes 😅 
```suggestion
    "data-h2-font-size": "base(caption)",
    "data-h2-padding": "base(x.25, x.75)",
```